x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Popover/Buttons.tsx

```diff
@@ -45,19 +44,7 @@ export function Buttons({
         </EuiButton>
       </EuiFlexItem>
       <EuiFlexItem>
-        <EuiButton
-          isDisabled={isAlreadyFocused}
-          color="secondary"
-          href={focusUrl}
-          onClick={onFocusClick}
-          title={
-            isAlreadyFocused
-              ? i18n.translate('xpack.apm.serviceMap.alreadyFocusedTitleText', {
-                  defaultMessage: 'Map is already focused'
-                })
-              : undefined
-          }
-        >
+        <EuiButton color="secondary" href={focusUrl} onClick={onFocusClick}>
```

Comment:
Why doesn't this get disabled any more? Now clicking it just does nothing.

Reply:
Er, I guess it centers the map on that element but that kind of changes the meaning of what "focus" is.

Reply:
The expectation of user flow is: click focus button -> navigate to and render map centered on a selected node. With this update, it flow is the same, except it skips the loading/navigation since it's already loaded. No matter when I click that button, the end state should be the same. At least that's what i felt after playing with it for a while. Having the button disabled felt more jarring.

Reply:
Seems ok I guess.